.github/workflows/deploy.yml

```diff
@@ -21,16 +21,20 @@ env:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    if: ${{ github.event_name != 'workflow_run' || github.event.workflow_run.conclusion == 'success' }}
```

Comment:
I still don't get how this works. About we have the following:

```
workflow_run:
  workflows:
    - Test
  branches:
    - master
    - development
  types:
    - completed
```

So by having this `github.event_name != 'workflow_run'`, we allow to run the deployment manually? And there is no automatic deployment to production since the default branch is `development`?

Reply:
Sorry. Let me explain more.
The `workflow_run` part is just used for workflow's events definition. So this workflow can be run when workflow `Test` finish on master and development branches .
About the `if` condition, I want to modify as we only need to check the status of previous workflow if it's `workflow_run`, when another action trigger this workflow, it should be run normally.
After this fix, the workflow `Deploy` can be called either when workflow `Test` finish successful or trigger manually from Github UI.